docs/concepts/linking-components/index.mdx

```diff
@@ -36,20 +36,20 @@ When running a component on wasmCloud, you don't have to worry about whether you
 
 In wasmCloud, components may be linked to...
 
-* [**Host functions**](/docs/concepts/linking-components/linking-at-runtime#host-functions) (Runtime): The wasmCloud host provides several functions available for import by components (for example, logging). 
+* [**Built-in providers**](/docs/concepts/linking-components/linking-at-runtime#built-in-providers) (Runtime): The wasmCloud host provides several functions available for import by components (for example, logging) through built-in providers. 
```

Comment:
```suggestion
* [**Built-in capability providers**](/docs/concepts/linking-components/linking-at-runtime#built-in-providers) (Runtime): The wasmCloud host provides several functions available for import by components (for example, logging) through built-in providers. 
```

So I'm not *sure* if we want to drop the "capability" in "capability providers" everywhere, so I'll only leave this here! Feel free to disregard if you think providers flow better/it's obvious